.dockerignore

```diff
@@ -8,4 +8,4 @@ node_modules/
 .svelte-kit/
 .env*
 !.env
-!.env.local
\ No newline at end of file
+.env.local
```

Comment:
@nsarrazin What is the recommended way to use `.env.local` with Docker? 

## Context
The [readme](https://github.com/huggingface/chat-ui/blob/93b39a0beb72378c76d5d146bfd3a8355c1d110d/README.md) is currently optimised for local envs and requires changes for Docker envs. 

## Issue
I stumbled upon the issue that the `.env.local` was ignored when building the Docker image in CI because of this ignore statement. 

## Possible Solutions
A temporary solution when using Docker Compose is available in PR https://github.com/huggingface/chat-ui/pull/1238, but it doesn't apply to Kubernetes deployments. Reverting this change—i.e., to `!.env.local`—would work for both Docker Compose and Kubernetes.

Reply:
I've added a few examples[here](https://github.com/huggingface/chat-ui/wiki/Deployment-examples#run-as-a-docker-image) but i'll be improving the docs for this.

Reply:
Furthermore, including `.env.local` in `Dockerfile` requires this change also:

```diff
+ COPY --chown=1000 .env.local /app/.env.local
```

Reply:
I faced several issues when deploying this to a Kubernetes cluster. I ended up passing somehow the `.envlocal` into the Docker image.